compiler/deepdive-default.conf

```diff
@@ -12,7 +12,7 @@ deepdive {
   }
 
   sampler.sampler_cmd: "sampler-dw"
-  sampler.sampler_args: "-l 1000 -s 1 -i 1000 --alpha 0.01"
+  sampler.sampler_args: "-l 1000 -s 1 -i 1000 --alpha 0.01 --sample_evidence"
```

Comment:
We are using it here because of the noisy rules. To be honest I don't think we should have it on by default but that's just my opinion.